kubernetes/INSTALL.md

```diff
@@ -7,11 +7,11 @@
     
     kubectl config set-context `kubectl config view | grep current-context | awk '{print $2}'` --namespace <ns>
 
-**Install Istio infra**
+**Install Istio control plane and management plane**
 
```

Comment:
While we can debate about control can management plane differences, the main architecture diagram shows a single control plane. Not two separate things. This makes it sound like we have too many moving parts. I suggest we stick to standard control plane, a terminology commonly used in the software defined networking and familiar to many.